### ISSUE TITLE
Part/device/package sequence divide

### ISSUE THREAD

**Issue by jakebeal** [iGEM-Engineering/iGEM-distribution#58]
Currently, when we say "J23101", the meaning is often muddled between three different notions:
 - The functional device, which is a promoter
 - The physical part, which embeds the promoter within an assembly context. That may be nothing, or it might include flanking sites, e.g., for BioBricks or various Type IIS
 - The vector package of the part, which might be null (e.g., a bare linear part DNA) or a plasmid. That plasmid also might package multiple part versions for the device, depending on the enzymes used to cut it out.

We need to be extremely clear about which things we are naming and which we are building.  I would suggest an analogy to the old TTL Databook, which included the same functional devices (e.g., Quadruple 2-input positive NAND gates was device 00) into many different families of packages with different characteristics (e.g., SN5400, SN75LS00).

My propose is as follows:
- The "parts and devices" sheet is actually specifying functional devices
- The "libraries and composite parts" sheet specifies the physical parts and packages

**Comment by vinoo-igem** [comment]
> We need to be extremely clear about which things we are naming and which we are building
Agreed! And there is a terminology divide here. I'm not advocating for this but iGEM currently uses different terms for these, and they also do not map fully.

**Comment by traci-igem** [comment]
> there is a terminology divide here

It sounds like it would be useful to create a dictionary of terms we want to use moving forward, so we're all in agreement about what we're talking about (i.e., what's a functional unit? what's a functional device? how do we define sample?). If we can create a short list of terms we need to define and start working on those definitions, then we can make the documentation / files that we're working in more clear.

**Comment by jakebeal** [comment]
Views of a part:
 - Functional part (free of assembly context) - doesn't always exist; this is what the current registry calls a part
 - Part in carrying vector - this is a sample in the distribution. The vector might be null (e.g., a linear fragment)
 - Part clipped out of vector
 - Part after assembly with other parts

Once a part has been assembled with others, that composite part will have the same four views as well.

In some cases views will be the same, in others they will not

**Comment by jakebeal** [comment]
A proposal for this has been made in: https://github.com/SynBioDex/SEPs/blob/master/sep_055.md
